src/instructlab/configuration.py

```diff
@@ -564,6 +578,11 @@ class Config(BaseModel):
         description="Configuration file structure version.",
         frozen=True,  # don't allow this to be changed anywhere in the code
     )
+    # prompts configuration
+    prompts: _prompts = Field(
```

Comment:
Since this field is only being used in chat, it should only be included in `_chat`.